installing.md

```diff
@@ -235,6 +235,25 @@ The other major section, `imports`, defines settings for each importer. The defa
 As you can see, the default datapaths are meant to be changed. This is also where you can enable
 admin lookup by overriding the default value.
 
+### Elasticsearch Configuration
+
+Of special note in `pelias.json` are Elasticsearch settings. The [default](https://github.com/pelias/config/blob/master/config/defaults.json#L19)
```

Comment:
Just call out the general `elasticsearch` section of the config, it's not a very big JSON blob.